app-misc/fhem/fhem-5.8.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit user
+
+DESCRIPTION="A GPL'd perl server for house automation"
+HOMEPAGE="https://www.fhem.de/"
+SRC_URI="https://www.fhem.de/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-2"
```

Comment:
Did you check if it is "GPL-2" or "GPL-2+"?

Reply:
The site https://fhem.de/fhem_DE.html states its "License: GPL (v2)" (at the bottom)

Reply:
It's code that matters, not homepage. https://svn.fhem.de/trac/browser/trunk/fhem/fhem.pl